shell/platform/windows/win32_window.h

```diff
@@ -0,0 +1,99 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_FLUTTER_WIN32_WINDOW_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_FLUTTER_WIN32_WINDOW_H_
+
+#include <Windows.h>
+#include <Windowsx.h>
+#include <memory>
+#include <string>
+
+#include "flutter/shell/platform/windows/win32_dpi_helper.h"
+
+namespace flutter {
+
+// A class abstraction for a high DPI aware Win32 Window.  Intended to be
+// inherited from by classes that wish to specialize with custom
+// rendering and input handling
+class Win32Window {
+ public:
+  Win32Window();
+  ~Win32Window();
+
+  // Initialize and show window with |title| and position and size using |x|,
+  // |y|, |width| and |height|
+  void Initialize(const char* title,
+                  const unsigned int x,
+                  const unsigned int y,
+                  const unsigned int width,
+                  const unsigned int height);
+
+  // Release OS resources asociated with window.
+  virtual void Destroy();
+
+ protected:
+  // Helper function to convert a c string to a wide unicode string.
```

Comment:
s/Helper function to convert/Converts/

Same below.

(The style guide strongly discourages comments that just say what the language construction itself indicate; it's obviously a function, and the fact that it's a "helper" is basically just a restatement of the fact that it's not public.

Reply:
Fixed in next commit